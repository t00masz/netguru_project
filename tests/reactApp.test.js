import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import ReactApp from '../src/reactApp'

test('should render', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<ReactApp />);
    expect((renderer.getRenderOutput())).toMatchSnapshot();
}) 
