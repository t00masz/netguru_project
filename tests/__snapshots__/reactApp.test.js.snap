// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="appContainer"
>
  <h1
    className="main-header"
  >
    Movie App
  </h1>
  <div
    className="movie-container"
  >
    <legend
      className="movie-container__label"
    >
      Save movie to database
    </legend>
    <input
      className="movie-container__title-input"
      name="title"
      onChange={[Function]}
      placeholder="Movie title (required)"
      type="text"
      value=""
    />
    <label
      className="movie-container__title-error"
      name="movieTittleError"
    >
       
      
       
    </label>
    <button
      className="movie-container_button-submit"
      name="movieSubmit"
      onClick={[Function]}
    >
      Send data!
    </button>
  </div>
  <div
    className="list-container"
  >
    <legend
      className="list-container__label"
    >
      List of all movies
    </legend>
    <p />
    <ul>
      <table
        className="list-container__table"
      >
        <thead>
          <tr>
            <th
              className="list-container__table-header"
              width="150px"
            >
              Title
            </th>
            <th
              className="list-container__table-header"
              width="50px"
            >
              Year
            </th>
            <th
              className="list-container__table-header"
              width="50px"
            >
              Rated
            </th>
            <th
              className="list-container__table-header"
              width="100px"
            >
              Released
            </th>
            <th
              className="list-container__table-header"
              width="50px"
            >
              Runtime
            </th>
            <th
              className="list-container__table-header"
              width="100px"
            >
              Genre
            </th>
            <th
              className="list-container__table-header"
              width="100px"
            >
              Director
            </th>
            <th
              className="list-container__table-header"
              width="150px"
            >
              Writer
            </th>
            <th
              className="list-container__table-header"
              width="100px"
            >
              Language
            </th>
            <th
              className="list-container__table-header"
              width="100px"
            >
              Country
            </th>
            <th
              className="list-container__table-header"
              width="100px"
            >
              Awards
            </th>
            <th
              className="list-container__table-header"
              width="400px"
            >
              Plot
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </ul>
    <label
      className="list-container__emptyMovieList-error"
      name="emptyMovieList"
    >
       
      
       
    </label>
  </div>
  <div
    className="comment-container"
  >
    <legend
      className="comment-container__label"
    >
      Save comment to database
    </legend>
    <select
      className="comment-container__selectTitle"
      name="movies"
      onChange={[Function]}
    />
    <input
      className="comment-container__comment-input"
      name="comment"
      onChange={[Function]}
      placeholder="Comment (required)"
      type="text"
      value=""
    />
    <label
      className="comment-container__comment-error"
      name="movieCommentTittleError"
    >
       
      
       
    </label>
    <p />
    <button
      className="comment-container_button-submitComment"
      name="movieCommentSubmit"
      onClick={[Function]}
    >
      Comment!
    </button>
  </div>
  <div
    className="commentList-container"
  >
    <legend
      className="commentList-container__label"
    >
      Show comments by associated movie
    </legend>
    <select
      className="commentList-container__selectComment"
      name="comments"
      onChange={[Function]}
    >
      <option
        value="Show all comments"
      >
        Show all comments
      </option>
    </select>
    <table
      className="commentList-container__table"
    >
      <thead>
        <tr>
          <th
            className="commentList-container__table-header"
            width="150px"
          >
            Title
          </th>
          <th
            className="commentList-container__table-header"
            width="650px"
          >
            Comment
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
    <label
      className="commentList-container__emptyCommentList-error"
      name="emptyCommentList"
    >
       
      
       
    </label>
  </div>
  <div
    className="footer-container"
  >
    <p />
  </div>
</div>
`;
